toptal/picasso
# feat: add grid support for radio and checkbox group

[SPT-1530](https://toptal-core.atlassian.net/browse/SPT-1530)

### Description

Adding grid support to the radio group and the checkbox group.

Instead of replacing the `horizontal` property with a `variant` property (which will add breaking changes), to display a grid-like radio group, you just need to pass the `small/medium/large` property to the radio /checkbox group.

To define space between radio/checkboxes you can use the `spacing` property.

### How to test

- Check [radio group grid](https://picasso.toptal.net/SPT-1530-add-grid-for-radio-and-checkbox-group/?path=/story/forms-radio--radio#radio-group-grid)
- Check [checkbox group grid](https://picasso.toptal.net/SPT-1530-add-grid-for-radio-and-checkbox-group/?path=/story/forms-checkbox--checkbox#checkbox-group-grid)

### Screenshots

![image](https://user-images.githubusercontent.com/2583281/115890158-4fa41800-a45d-11eb-89ca-589fa2883908.png)

![image](https://user-images.githubusercontent.com/2583281/115896317-f68bb280-a463-11eb-998a-d29b684c6eaf.png)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
